secant/Util/InitialisationState.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  InitialisationState.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 30.03.2022.
+//
+
+import Foundation
+
+enum InitialisationState: Equatable {
+    case failure(ErrorWrapper)
+    case initialized
+    case keysMissingFilesPresent
```

Comment:
I think you can just call out what is missing here.  Because when you have both missing it's "uninitialized" then when either one of these is missing, you can just report on the missing piece.
i.e. `keysMissing` and `databaseMissing`

Reply:
fixed